src/main/scala/firrtl/passes/Uniquify.scala

```diff
@@ -56,6 +56,7 @@ import firrtl.Mappers._
   *      to rename a
   */
 object Uniquify extends Pass {
+  import MemPortUtils.memType
```

Comment:
Could also just not import it and use MemPortUtils.memType as the function call below